taca/server_status/cli.py

```diff
@@ -13,13 +13,54 @@ def server_status():
 		raise RuntimeError("Configuration missing required entries: server_status")
 
 
-
 # server status subcommands
 @server_status.command()
-@click.option('--credentials', type=click.Path(exists=True), default=os.path.join(os.path.dirname(__file__), 'gdocs_credentials.json'),
+@click.option('--gdocs', is_flag=True, default=True, help="Update the google docs")
```

Comment:
How does this work? If it defaults to `True`, what happens if it is specified? Is there any way to _not_ update google docs?